src/main/java/step3/domain/Car.java

```diff
@@ -1,32 +1,26 @@
 package step3.domain;
 
 public class Car {
-	private final int MIN_MOVE_VALUE = 4;
-	private final int speed = 1; // 나중에 speed 설정하게 되면 final 제거 및 생성자 추가 필요
-	private int position;
-	private int moveCalledCount;
+	public static int DEFAULT_SPEED = 1;
+	public static int STOP_SPEED = 0;
+
+	private Integer position;
+	private CarMoveStrategy moveStrategy;
 
 	public Car() {
 		this.position = 0;
-		this.moveCalledCount = 0;
-	}
-
-	public void move(int input) {
-		if (input >= MIN_MOVE_VALUE) {
-			moveForward();
-		}
-		this.moveCalledCount++;
+		this.moveStrategy = new CarAtDefaultStrategy(); // 기본 전략 지정
 	}
 
-	public void moveForward() {
-		this.position = this.position + this.speed;
-	}
-
-	public int getMoveCalledCount() {
-		return moveCalledCount;
+	public void move(Integer randVal) {
+		this.position += this.moveStrategy.getMovePosition(randVal);
```

Comment:
randVal도 임의의 값 문자열에 의한 전진조건 평가인데 이러면 전진전략의 유연함과 확장성이 Int타입의 숫자로만 제한되어 의도한만큼의 확장성은 나오지 않을 것 같네요. 🤔  다음과 같은 방식도 고민해보세요!
```suggestion
	public void move() {
		if (movable()) { position += speed; }
	}
	.
	.
	.
	public boolean movable() {
	    return moveStrategy.movable();
	}	
		
```